cea/functions.py

```diff
@@ -2,12 +2,14 @@
 from __future__ import division
 
 import math
-import numpy as np
 import os
+
+import numpy as np
 import pandas as pd
 import scipy
 import scipy.optimize as sopt
 
+import contributions.reporting as reporting
```

Comment:
please don't import from contributions or sandbox... hm... ok. I see your problem - we have to remove that when merging the pull request back into core...